.github/workflows/test.yaml

```diff
@@ -8,6 +8,8 @@ jobs:
         python: [ 3.11, 3.12 ]
         os: [ ubuntu-latest, windows-latest ]
     runs-on: ${{ matrix.os }}
+    env: 
+      PYTHONUTF8: 1
```

Comment:
🤦🏻 I swear I scoured stack overflow looking for a way to force UTF-8 in the runner. That was surprisingly easy... Thank you so much